### DISCUSSION TITLE
Строку как ссылку открывать с https://

### DISCUSSION THREAD

**Discussion by nikidjan** [totumonline/totum-ru-issues-and-discussions#504] [Как это сделать?] [answered]
Можно ли ка кто сделать что бы ссылка открывалась не по адресу хост/ссылка а по https://ссылка
Например у меня таблица "Домены" и я записываю их туда без https:// и в итоге они открываются как внутренние страницы
Временно придумал сделать еще одно поле "кнопка", назвал её "www ", в ней код что бы ею открывать ссылки, но как то не изящно )
Если системно нет решения, может подскажете как еще можно реализовать?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Вот так в этом поле:

<img width="1155" alt="Снимок экрана 2024-01-08 в 18 46 32" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/a48814dd-6707-4978-a2de-ff226b45b91c">

**Reply by ttm-alexeypolunin** [reply]
Стандартную настройку Открывать как URL у поля Строка надо убрать в этом случае, а то мне каж она перекрывает или они подерутся.

**Reply by nikidjan** [reply]
Красота! Большое спасибо!

**Reply by nikidjan** [reply]
а у селектов ссылка если не открывает ничего и в логе ничего тоже не пишет - что то не так сделал? или все таки для селектов ссылку таким образом не сделать?
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/126777661/22790abc-3e79-421c-b6bb-89dfa10abac5)

**Reply by ttm-alexeypolunin** [reply]
Это у нас ошибка — там именно для Селекта судя по всему клик оформленный как url перекрывается с установкой пойнтера.
